crates/meroctl/src/cli/app/install.rs

```diff
@@ -13,13 +17,21 @@ use crate::config_file::ConfigFile;
 #[derive(Debug, Parser)]
 pub struct InstallCommand {
     /// Path to the application
-    #[arg(long, short)]
-    pub path: Utf8PathBuf,
+    #[arg(long, short, conflicts_with = "url")]
+    pub path: Option<Utf8PathBuf>,
+
+    /// Url of the application
+    #[clap(long, short, conflicts_with = "path", requires = "metadata")]
+    pub url: Option<String>,
```

Comment:
Lovely to see some Rustdocs being written! 😄 👍 I'm hoping that it can happen everywhere... it would be good to add them to everything new that gets added 🙂

Reply:
> Lovely to see some Rustdocs being written!

these aren't just docs for rust docs though, as they serve dual-purpose and it's the second that pretty much mandates their presence, given that they're visible to the users of the CLI

Ideally, we prioritize documenting all public interfaces and work our way in over time.